cmd-record.c

```diff
@@ -352,9 +354,46 @@ static int fill_file_header(struct opts *opts, int status, struct rusage *rusage
 	if (write(fd, &hdr, sizeof(hdr)) != (int)sizeof(hdr))
 		pr_err("writing header info failed");
 
+	char *orig_args = opts->args;
+	char *orig_retval = opts->retval;
+	if (opts->auto_args) {
+		/* add auto-args info to write into info file */
+		size_t alloc_size;
+
+		if (orig_args) {
+			alloc_size = strlen(auto_args_list) + strlen(orig_args) + 2;
+			opts->args = (char*)malloc(alloc_size);
+			sprintf(opts->args, "%s;%s", auto_args_list, orig_args);
+		}
+		else {
+			alloc_size = strlen(auto_args_list) + 1;
+			opts->args = (char*)malloc(alloc_size);
+			strncpy(opts->args, auto_args_list, alloc_size);
+		}
+		pr_dbg("opts->args = %s\n", opts->args);
```

Comment:
For the debug message, I think a level 1 message (`pr_dbg`) should be used to describe high level actions.  I don't want to too many messages when using `-v` alone.

In this case, it may be like "auto arg is used" or "extending args using builtin auto-arg list".  Messages like above can use level 2 or 3 according to the verbosity.

Reply:
Thanks. I changed it to `pr_dbg2`.